### ISSUE TITLE
auto-increment not working with SQLite 

### ISSUE THREAD

**Issue by arkceajin** [sequelize/sequelize-auto#209]
I created an SQLite database using Navicat. But the `sequelize-auto` is not able to generate the auto-increment for the primary key. I export the table into `.sql` like this:
```
PRAGMA foreign_keys = OFF;

-- ----------------------------
-- Table structure for users
-- ----------------------------
DROP TABLE IF EXISTS "main"."users";
CREATE TABLE "users" (
"id"  INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
"user_name"  TEXT(50) NOT NULL,
"password"  TEXT(128) NOT NULL
);
```
Generated `sequelize ` schema by `sequelize-auto` :
```
module.exports = function(sequelize, DataTypes) {
	return sequelize.define('users', {
		id: {
			type: DataTypes.INTEGER,
			allowNull: false,
			primaryKey: true
		},
		user_name: {
			type: DataTypes.TEXT,
			allowNull: false
		},
		password: {
			type: DataTypes.TEXT,
			allowNull: false
		}
	}, {
		tableName: 'users'
	});
};

```
Using `sequelize-auto  -h localhost -d mysqlite.db -e sqlite -c ./sqlite-option.json -o "./schema/"`
didn't generate the `autoIncrement: true`,
I have to add it by myself. So is this a bug? or I missed something?

**Comment by ericwastaken** [comment]
I'm observing the same behavior on a sqlite DB where I have autoIncrement fields.

**Comment by eladbenami** [comment]
same issue. when it will be solved?

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.2`
